sdk/tables/azure-data-tables/azure/data/tables/_deserialize.py

```diff
@@ -158,12 +158,12 @@ def _convert_to_entity(entry_element):
 
     # Partition key is a known property
```

Comment:
Could we add a small `TODO` comment here to investigate whether we can entirely remove lines 159-167? I'm not entirely sure what this achieves....

Reply:
Updated